src/mainwindow.cpp

```diff
@@ -105,13 +105,14 @@ bool MainWindow::event(QEvent* event)
 void MainWindow::resizeEvent(QResizeEvent* event)
 {
    QMainWindow::resizeEvent(event);
-   updateImage(true);
+   this->findChild<QLabel*>("image")->clear();
```

Comment:
will this not result in a fading animation when the resize event is fired?

Reply:
No, because of `oldImage` will be `NULL` in 
https://github.com/NautiluX/slide/pull/42/files/680ad9b99a05e8f3a144bfb1c2ede6c415573475#diff-c72b037f3ac7abbd3d0ee519b6f456013a9bbf5a59df16308939320c04970447L159